src/rgw/rgw_admin.cc

```diff
@@ -5763,6 +5768,11 @@ int main(int argc, const char **argv)
   if (admin_specified)
      user_op.set_admin(admin);
 
+  if (reader_specified) {
+     user_op.set_admin(reader);
```

Comment:
Is this feature meant to set read rights on all kinds of users?

Reply:
I thought it did. Maybe I don't understand what users are local then. I thought those that are serialized and written into a bucket were local.